### DISCUSSION TITLE
VLLM output not complete

### DISCUSSION THREAD

**Discussion by RickyGunawan09** [vllm-project/vllm#1095] [Q&A]
hai guys,
thank you for making this super library.
i have a question about the output of vllm

i'm using GPU RTX A6000 50GB cuda 12 with model Vicuna13B-v1.5-4k from lmsys
vllm is serve with gpu_memory_utilization 0.8
the parameter that i change for request is:

max_token 4096
temperature 0
i'm make custom prompt with context from text/document.

why sometimes the output is not complete ?

**Comment by zhiqiangzhongddu** [comment]
+1, my obtained answers are always not complete. Even shorter than a sentence.

**Comment by zhiqiangzhongddu** [comment]
Hi this might be helpful for you. You can set the length of output to get complete answers. https://github.com/vllm-project/vllm/blob/ee8217e5bee5860469204ee57077a91138c9af02/vllm/sampling_params.py#L61

**Reply by jayachandrakalakutagar** [reply]
Still broken responses
